simplex.py

```diff
@@ -195,25 +197,20 @@ def run(self, r_data=None):
             ref_ff = self.ff.__class__()
             ref_ff.method = 'REFLECTION'
             ref_ff.params = copy.deepcopy(best_ff.params)
+            # Need score difference sum for weighted inversion. If zero, should break
+            score_diff_sum=sum([x.score - self.new_ffs[-1].score
```

Comment:
PEP8 is new to me, but I don’t mind learning. Do you make the changes when you see them, or should I make them and re-commit? I think you complained about that when we found small things and wanted you to do it?

/Per-Ola

From: Eric Hansen [mailto:notifications@github.com]
Sent: den 28 juni 2016 16:16
To: ericchansen/q2mm <q2mm@noreply.github.com>
Cc: Norrby, Per-Ola <Per-Ola.Norrby@astrazeneca.com>; Author <author@noreply.github.com>
Subject: Re: [ericchansen/q2mm] Mysimplex (#36)


In simplex.py<https://github.com/ericchansen/q2mm/pull/36#discussion_r68764511>:

> @@ -195,25 +197,20 @@ def run(self, r_data=None):

>              ref_ff = self.ff.__class__()

>              ref_ff.method = 'REFLECTION'

>              ref_ff.params = copy.deepcopy(best_ff.params)

> +            # Need score difference sum for weighted inversion. If zero, should break

> +            score_diff_sum=sum([x.score - self.new_ffs[-1].score

Using an equals sign without spaces is used for arguments for a function, not for declaring variables. This is standard PEP8<http://legacy.python.org/dev/peps/pep-0008/#other-recommendations>.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/ericchansen/q2mm/pull/36/files/0b9c5d0ad0a8d1105e4d26f192324c8bf323a928..398b4e1b8e1242703edfcc5b2a40b20047233942#r68764511>, or mute the thread<https://github.com/notifications/unsubscribe/ANVSNHMUSZiL5bcRItEAaGX5KZcT2K9Zks5qQSyogaJpZM4JAC7r>.

________________________________

Confidentiality Notice: This message is private and may contain confidential and proprietary information. If you have received this message in error, please notify us and remove it from your system and note that you must not copy, distribute or take any action in reliance on it. Any unauthorized use or disclosure of the contents of this message is not permitted and may be unlawful.